src/content/staking/saas/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Staking as a service
+description: An overview of how to get started with pooled ETH staking
+lang: en
+template: staking
+emoji: ":money_with_wings:"
+sidebar: true
+image: ../../../assets/staking/leslie-saas.png
+alt: Leslie the rhino floating in the clouds.
+sidebarDepth: 2
+summaryPoints:
```

Comment:
I think a 3rd bullet point could be the value prop that while you're trusting a service with your signing keys, you're able to maintain sole access of your withdrawal keys, which provides some assurances on your funds staying protected.